src/ucp/proto/rndv.c

```diff
@@ -747,9 +798,8 @@ ucp_rndv_send_frag_get_mem_type(ucp_request_t *sreq, uintptr_t rreq_ptr,
     freq->send.rndv_get.rkey           = rkey;
     freq->send.rndv_get.remote_address = remote_address;
     freq->send.rndv_get.remote_request = rreq_ptr;
-    freq->send.rndv_get.lanes_map      = 0;
-    freq->send.rndv_get.lane_count     = 0;
     freq->send.rndv_get.rreq           = sreq;
+    ucp_rndv_req_init_get_zcopy_lane_map(freq);
```

Comment:
do you really need to re-calculate it for every fragment?

Reply:
I think so because each fragment is tracked in separate rndv frag req

Reply:
can't we cache it somewhere, because it is supposed to be the same for all fragments?

Reply:
fixed. can you check it?

Reply:
@brminich can you please check?